style.css

```diff
@@ -3,10 +3,20 @@
   padding: 0;
 }
 
+a {
+  text-decoration: none;
+}
+
+ul li {
+  list-style: none;
+}
+
 body {
   position: absolute;
+  width: auto;
   height: 4900px;
   font-family: 'Roboto', sans-serif;
+  overflow-x: hidden;
 }
```

Comment:
- [ ] I recommend to delete the fixed height from the body. Currently there is white space at the end of the document, it will be gone once you delete the fixed height 😉
<img width="837" alt="Screenshot 2022-02-26 at 17 09 03" src="https://user-images.githubusercontent.com/77038610/155844501-148fb55d-f70a-4d2a-bfc7-77df3253f7fe.png">

Reply:
Thank you for the commendation. 
The observation is noted and the corrections made immidately.